roles/cri-o/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+crio_rhel_repo_base_url: 'https://cbs.centos.org/repos/paas7-openshift-origin311-candidate/x86_64/os/'
```

Comment:
why does it uses openshift origin ?

Reply:
Because it provide cri-o package. I haven't found other convenience package.